### PR TITLE
Added a menu to choose different games

### PR DESCRIPTION
**Pull request by Jeronics**
I added a main menu from which I can choose two games:
- guess the saint
- guess the painter

### CODE REVIEW

File: PaintingTrivia.py [hunk 1]
```diff
@@ -119,14 +128,49 @@ def show_entries():
             'selected_painter_id'] else 'wrong_guesses'
         session[key] += 1
 
-    painters = Painter.query.order_by(func.random()).limit(4).all()
+    while True:
+        painters = Painter.query.order_by(func.random()).limit(4).all()
+        selected_painter = random.choice(painters)
+        selected_painting = Painting.query.filter(Painting.painter == selected_painter).order_by(func.random()).limit(
```
Comment (denisergashbaev) [inline PaintingTrivia.py]: Why random again?

File: PaintingTrivia.py [hunk 2]
```diff
@@ -119,14 +128,49 @@ def show_entries():
...
+            1).first()
+        if selected_painting:
+            break
```
Comment (denisergashbaev) [inline PaintingTrivia.py]: Why  do u have while/true/break structure here?

File: PaintingTrivia.py [hunk 3]
```diff
@@ -119,14 +128,49 @@ def show_entries():
...
+            'selected_painter_id'] else 'wrong_guesses'
+        session[key] += 1
 
-    selected_painter = random.choice(painters)
+    while True:
+        painters = Painter.query.order_by(func.random()).limit(4).all()
+        selected_painter = random.choice(painters)
+        selected_painting = Painting.query.filter(Painting.painter == selected_painter).order_by(func.random()).limit(
+            1).first()
+        if selected_painting:
```
Comment (denisergashbaev) [inline PaintingTrivia.py]: Here too, i dont get the reason to use while and break here

File: database/users.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+from settings import db
+from models.user import User
+import crypt
+
+
+def add_user(user_name, user_password):
```
Comment (denisergashbaev) [inline database/users.py]: I have actually deleted this file in the branch u've merged into master. All the stuff from here is inside of the user object. I think we can safely delete this file

**Comment by denisergashbaev** [comment]
@Jeronics take a look at my comments please. I believe you habe not merged the master into your branch properly as there are some old code remaints floating around. Lets do that together tomorrow

File: PaintingTrivia.py (hunk 1 above)
Comment (Jeronics) [inline PaintingTrivia.py]: You do need random here, the game has to show random images!

File: PaintingTrivia.py (hunk 2 above)
Comment (Jeronics) [inline PaintingTrivia.py]: Some painters don't have paintings YET, this could happen at any given moment

File: PaintingTrivia.py (hunk 3 above)
Comment (Jeronics) [inline PaintingTrivia.py]: Same as before, some painters don't have paintings yet!

File: database/users.py (hunk 4 above)
Comment (Jeronics) [inline database/users.py]: I'm on it!

File: PaintingTrivia.py (hunk 2 above)
Comment (denisergashbaev) [inline PaintingTrivia.py]: then a better solution would be to modify the query to only return the painters that have at least one picture.. we can do it tonight...

Comment (Jeronics) [inline PaintingTrivia.py]: Yes, I am thinking about how to do it.. this was kind of a quick fix, we'll discuss it tonight!

Comment (denisergashbaev) [inline PaintingTrivia.py]: smth along these lines:
http://stackoverflow.com/questions/6044309/sqlalchemy-how-to-join-several-tables-by-one-query

Comment (denisergashbaev) [inline PaintingTrivia.py]: smth along these lines: http://stackoverflow.com/questions/6044309/sqlalchemy-how-to-join-several-tables-by-one-query

**Pull request merged**